### PR TITLE
suggested locale fixes

### PR DESCRIPTION
**Pull request by stroucki**
(copied from original)

This addresses the Spanish part of #3764 and two instances of non-ascii apostrophes that caught my eye. Also, some untranslated German messages are translated.

### CODE REVIEW

**Comment by tboerger** [comment]
This PR needs to be rebased. Beside that I'm fine with adding the translations here until we get up some translation system. LGTM

**Comment by strk** [comment]
In that case then I'm also ok. LGTM (but I cannot speak DE so don't really know)

File: conf/locale/locale_de-DE.ini [hunk 1]
```diff
@@ -915,9 +915,9 @@ dashboard.current_goroutine=Aktuelle Goroutines
 dashboard.current_memory_usage=Aktuelle Speichernutzung
 dashboard.total_memory_allocated=Zugeteilter Gesamtspeicher
 dashboard.memory_obtained=Erhaltener Speicher
-dashboard.pointer_lookup_times=Pointer Lookup Times
-dashboard.memory_allocate_times=Memory Allocate Times
-dashboard.memory_free_times=Memory Free Times
+dashboard.pointer_lookup_times=Zeigerlookup-Dauern
```
Comment (tboerger) [inline conf/locale/locale_de-DE.ini:918]: `Dauern`? I think it's `Dauer`. Same for the below.

**Comment by tboerger** [comment]
> In that case then I'm also ok. LGTM (but I cannot speak DE so don't really know)

I am german ;)

**Comment by tboerger** [comment]
Ah and it still needs to be rebased.

**Comment by tboerger** [comment]
@stroucki please rebase this PR, otherwise we need to move it to 1.1.0.

**Comment by metalmatze** [comment]
Hey @stroucki yet another one asking you to rebase. Would love to merge this! 😊

File: conf/locale/locale_de-DE.ini (hunk 1 above)
Comment (stroucki) [reply conf/locale/locale_de-DE.ini:918]: Dauer in the plural is Dauern, though it is technical use only. Quoting from my Duden:
Dauer, die |Dauer|
die Dauer; Genitiv: der Dauer, (FACHSPRACHE:) Dauern

Comment (stroucki) [reply conf/locale/locale_de-DE.ini:918]: Though if the value displayed is an average and not a series, the singular should be better (in English too).

**Comment by stroucki** [comment]
Anything I should be careful about moving between gogs and gitea?

**Comment by stroucki** [comment]
moved to #298

File: conf/locale/locale_de-DE.ini (hunk 1 above)
Comment (tboerger) [reply conf/locale/locale_de-DE.ini:918]: I am German and Dauern sounds wrong to me ;)

**Pull request closed without merging**